src/librustc/mir/mod.rs

```diff
@@ -1834,6 +1816,11 @@ pub enum FakeReadCause {
     /// generate a read of x to check that it is initialized and safe.
     ForMatchedPlace,
 
+    /// [Fake borrows](BorrowKind::Guard) don't conflict with existing borrows,
```

Comment:
does ... does this style of hyperlink *work* in rustdoc? I don't recall seeing it in API doc comments  before, but I also am not steeped in rustdoc features...

Reply:
This is #43466

Reply:
cool!